Cargo.toml

```diff
@@ -191,7 +191,7 @@ prost = { version = "0.10.4", default-features = false, features = ["std"] }
 prost-types = { version = "0.10.1", default-features = false, optional = true }
 
 # GCP
-goauth = { version = "0.13.0", default-features = false, optional = true }
+goauth = { version = "0.13.0", optional = true }
```

Comment:
I think @blt got us on the habit of not using the default features. I'm curious if he has thoughts here. Obviously not using them can cause issues like this, but using them can also pull in more features than we actually need.

Reply:
In this case, `default-features = false` would have to be followed by `features = ["default"]` which seems pretty nonsensical, as the feature enabled by the default features is a sub-crate feature, which can't be turned on directly by us.

Reply:
> In this case, default-features = false would have to be followed by features = ["default"]

Haha

> which seems pretty nonsensical, as the feature enabled by the default features is a sub-crate feature, which can't be turned on directly by us.

Agreed. I think this is the right move here with that context.